src/components/Button.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+class Button extends React.Component{
+
+    constructor(props){
+        super(props);
+    }
+
+    render(){
+        return(
+        <button>{this.props.name}</button>
+        )
+    }
+}
```

Comment:
the `name` prop should be a string, validate this with `propTypes`
[ see [requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/button/acceptProp.png) ]